src/OpenTelemetry/Trace/SimpleExportActivityProcessor.cs

```diff
@@ -23,70 +23,27 @@
 namespace OpenTelemetry.Trace
 {
     /// <summary>
-    /// Implements simple activity processor that exports activities in OnEnd call without batching.
+    /// Implements activity processor that exports <see cref="Activity"/> at each OnEnd call.
     /// </summary>
-    public class SimpleExportActivityProcessor : ActivityProcessor
+    public class SimpleExportActivityProcessor : ReentrantExportActivityProcessor
     {
-        private readonly ActivityExporterSync exporter;
-        private bool stopped;
+        private readonly object lck = new object();
```

Comment:
nit: Looks like "ick" :smile: Probably syncObject or lockObject is more common?

Reply:
I'm not a C# developer, borrowed/learned [this](https://github.com/open-telemetry/opentelemetry-dotnet/blob/e43356abd72b451af132018d49e5d23e1784b0ef/src/OpenTelemetry/Metrics/MeterProviderSdk.cs#L30) from @cijothomas 😺

Reply:
#1084

Reply:
:D Thanks for fixing the names throughout.